vector/src/main/java/im/vector/app/features/location/LocationSharingServiceConnection.kt

```diff
@@ -51,7 +52,12 @@ class LocationSharingServiceConnection @Inject constructor(
         callback = null
     }
 
+    fun stopLiveLocationSharing(roomId: String) {
+        locationSharingService?.stopSharingLocation(roomId)
+    }
+
     override fun onServiceConnected(className: ComponentName, binder: IBinder) {
+        locationSharingService = (binder as LocationSharingService.LocalBinder).getService()
```

Comment:
Should we reset `locationSharingService` to `null` in `onServiceDisconnected` for safety reason?

Reply:
Good catch, done.